src/util/std_types.h

```diff
@@ -1702,34 +1535,28 @@ class complex_typet:public type_with_subtypet
   }
 };
 
-/*! \brief Cast a generic typet to a \ref complex_typet
- *
- * This is an unchecked conversion. \a type must be known to be \ref
- * complex_typet.
- *
- * \param type Source type
- * \return Object of type \ref complex_typet
- *
- * \ingroup gr_std_types
-*/
+/// \brief Cast a generic typet to a \ref complex_typet
+///
+/// This is an unchecked conversion. \param type must be known to be \ref
+/// complex_typet.
+///
+/// \param type Source type.
+/// \return Object of type \ref complex_typet.
 inline const complex_typet &to_complex_type(const typet &type)
 {
   PRECONDITION(type.id()==ID_complex);
   return static_cast<const complex_typet &>(type);
 }
 
-/*! \copydoc to_complex_type(const typet &)
- * \ingroup gr_std_types
-*/
+/// \copydoc to_complex_type(const typet &)
 inline complex_typet &to_complex_type(typet &type)
 {
   PRECONDITION(type.id()==ID_complex);
   return static_cast<complex_typet &>(type);
 }
 
-/*! \brief A type for mathematical functions (do not
-           confuse with functions/methods in code)
-*/
+/// A type for mathematical functions (do not confuse with functions/methods
```

Comment:
In your follow up PR you might like to pull this one and `complex_typet` into a `mathematic_types.h` since they are quite special case.